pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
+> Try also `ugrep --help {{something}}`.
```

Comment:
I am not sure whether it's a good idea to place `--help` example here (what about to move it to the bottom?). I think tldr page format must be changed in order to allow add some samples for `--help`/`--version` options. For instance we can increase allowed code sample count or allow such lines for all pages after command description. On the other hand such options are common for many commands (but not for all).

Reply:
The `--help` option is different than usual in that it shows specific help for `{{something}}`. So I agree it is not common to see it at the top of the tldr page. I doubted adding this too, but also thought that putting it in a code sample takes away something that is "actionable" and not just intended for help. It is more of a thing that is complementary to tldr and a man page. So I'm not sure what to do either.

Reply:
In this case it's perhaps base to put it as an example if it's important.